makerdao-bridge-invariant/src/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { FindingSeverity, FindingType, ethers, getAlerts } from 'forta-agent';
+
+export type FindingInput = {
+  name: string;
+  description: string;
+  alertId: string;
+  protocol?: string;
+  severity: FindingSeverity;
+  type: FindingType;
+  metadata: {
+    [key: string]: string;
+  };
+};
+
+export const getNormalizedAmount = (amount: ethers.BigNumber, decimals: number) => {
+  const divValue =
+    decimals > 12 ? ethers.BigNumber.from(10 ** 12).mul(10 ** (decimals - 12)) : ethers.BigNumber.from(10 ** decimals);
```

Comment:
If the reason was the overflow, there's this simpler way that also works:

```suggestion
  const divValue = ethers.BigNumber.from((10 ** decimals).toString());
```

Reply:
Oh Yes, I could have just done .toString()!!